xlators/mgmt/glusterd/src/glusterd-locks.c

```diff
@@ -524,8 +524,7 @@ glusterd_mgmt_v3_lock(const char *name, uuid_t uuid, uint32_t *op_errno,
     glusterd_conf_t *priv = NULL;
     gf_boolean_t is_valid = _gf_true;
     uuid_t owner = {0};
-    xlator_t *this = NULL;
-    char *bt = NULL;
+    xlator_t *this = THIS;
```

Comment:
Are lines 533-534 still needed?

Reply:
No. But since, this is a backport I didn't make additional changes to the PR.